ome_zarr/format.py

```diff
@@ -124,4 +125,15 @@ def init_store(self, path: str, mode: str = "r") -> FSStore:
         return store
 
 
-CurrentFormat = FormatV02
+class FormatV03(FormatV02):  # inherits from V02 to avoid code duplication
+    """
+    Changelog: variable number of dimensions (up to 5),
+    introduce axes field in multiscales (June 2021)
+    """
+
+    @property
+    def version(self) -> str:
+        return "0.3"
```

Comment:
Is there _no_ v03 specific logic that can be encapsulated here?

Reply:
I am not quite sure what you mean by this.

Reply:
Perhaps as with https://github.com/ome/ome-zarr-py/pull/89/files/41a6c6a95446014c6ea409df6fb823cd6a39f8bb#r665674289, this is roughly regarding what the `Format` class hierarchy should be responsible for, but I don't think it's a blocker, it just surprised me that there was no logic in the class.

Reply:
Ok, I think this is a larger design question. I suggest we move ahead without touching on this now.